articles/quickstart/backend/java-spring-security5/interactive.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Spring Boot API
+description: Secure your API using Spring Security 5 and Auth0
+interactive: true
+alias:
+  - spring security
+  - spring
+topics:
+  - quickstart
+  - backend
+  - spring
+files:
+  - files/application
+  - files/audience-validator
+  - files/security-config
+  - files/message
+  - files/api-controller
+github:
+  path: 01-Authorization-MVC
+---
+
+# Secure your API using Spring Security 5 and Auth0
+
+Auth0 allows you to quickly add authentication and gain access to user profile information in your application. This guide demonstrates how to integrate Auth0 with any new or existing Spring Boot application using the [auth0-spring-security-api](https://github.com/auth0/auth0-spring-security-api) package.
```

Comment:
Should we talk about Authorization here instead of Authentication and User Profile Information?

Reply:
We are not doing that here Fred because the authorisation functionality is currently provided by the spring framework itself `.hasAuthority("SCOPE_read:messages")`. Auth0 provides scope for authorisation and Spring framework does the authorisation. Our SDK doesn't play a role here

Reply:
What I mean is

> Auth0 allows you to quickly add authentication and gain access to user profile information in your application. 

Should probably be

> Auth0 allows you to quickly add authorization to your application. 


In an API scenario, we talk about Authorization, not Authentication.